handbook/engineering/incidents/playbooks/index.md

```diff
@@ -97,13 +97,30 @@ watch -n1 kubectl get all -n prod -l app=gitserver -o wide
 ```
 
 ### 4. Verify service is restored
+
 Open the alert UI to click on the check URL that was failing and verify it's now working again.
 
+## Restarting pods
+
+This action is also referred to as "bouncing pods" or to "bounce a pod". Follow the steps to [authenticate kubectl](../../deployments.md) and perform a restart using kubectl to restart the desired deployment:
+
+```sh
+kubectl rollout restart deployment/$SERVICE
+```
+
+Note the use of `deployment/` - you cannot restart services (`svc/`). Note that [stateful sets have a different process](#making-updates-to-stateful-sets).
```

Comment:
```suggestion
Note the use of `deployment/` - you cannot restart services (`svc/`). Note that [stateful sets have a different process](#making-updates-to-stateful-sets).

You can also bounce pods by manually finding them via `kubectl get pods` and `kubectl delete pod $POD_ID`. Once a pod is deleted, it will automatically be recreated.
```